src/auto-trace.helper.spec.js

```diff
@@ -180,6 +205,32 @@ describe('auto-trace.js', () => {
   at fancyUpdateAnswer (../src/source-forms/questions/question.component.js:77:3)`;
 			expect(removeAutoTraceFromErrorStack(err).stack).toEqual(expectedStack);
 		});
+		it('should remove self regardless of the file', () => {
+			//IE and Edge docs - https://msdn.microsoft.com/en-us/library/windows/apps/hh699850.aspx
+			const err = new Error('err');
+			err.stack = `Error: it broke
+	at catchAsyncStacktrace(/global-settings.js:1929:28)
```

Comment:
This line should be removed regardless of the file name `(/global-settings.js:1929:28)`